integration_tests/cli/auth-demo.sh

```diff
@@ -265,8 +291,191 @@ echo "Get $user account"
 result=$(dcld query auth account --address=$user_address)
 check_response "$result" "Not Found"
 
+
+test_divider
+
+echo "Jack proposes account for $user"
+result=$(echo $passphrase | dcld tx auth propose-add-account --info="Jack is proposing this account" --address="$user_address" --pubkey="$user_pubkey" --roles="Vendor" --vid="$vid" --from jack --yes)
+check_response "$result" "\"code\": 0"
+
 test_divider
 
+echo "Get list revoked account must be exist"
```

Comment:
Message should be `Get all revoked accounts. $user account in the list` to be consistent

Reply:
Done! ✅